### PR TITLE
Add APM HOL with media images

### PR DESCRIPTION
**Pull request by sawpresto**
Update image files to be less than 150 kb :)

### CODE REVIEW

File: docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+#Application Performance Monitoring - PartsUnlimited
+
+The DevOps team has noticed that the recommendations page is slow to load and shows performance spikes in Application Insights telemetry. By viewing the details of performance monitoring through Application Insights, we will be able to drill down to the code that has affected the slow performance of the web application and fix the code.
+
+In this lab, you will learn how to set up Application Insights telemetry, and drill down into performance monitoring data through Application Insights in the new Azure Portal.
+
+**Prerequisites**
+
+- Visual Studio 2013 or higher
```
Comment (EMaher) [inline docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md]: Is this HOL for the website based on ASP.NET 4.5 or ASP.NET 5?  If it's ASP.NET 5 then I think you need VS 2015.

File: docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+- PartsUnlimited website deployed to a Microsoft Azure Web App (see [link](https://github.com/Microsoft/PartsUnlimited/blob/master/docs/Deployment.md))
```
Comment (EMaher) [inline docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md]: If this HOL if for 4.5, please change link to point to the aspnet45 branch, just to make things clear.

**Comment by EMaher** [comment]
@sawpresto 

Is this HOL only written for the ASP.NET 4.5 version of the Website in the aspnet45 branch?  All my other comments depend on the answer to this question.  Please let me know and then I'll continue the code review.  Thanks.

**Comment by sawpresto** [comment]
@EMaher Yes, this is written for the aspnet45 branch. I'm just pushing the changes up to have the deployment link from master to aspnet45, I'll update the branch.

**Comment by EMaher** [comment]
@sawpresto Excellent.  Thanks for the quick reply.  I'll start looking at the rest of the updates in the meantime.

**Comment by sawpresto** [comment]
Awesome. I think that it's already pushed up ("Change deployment link from master to aspnet45") right before your comment 5 minutes ago. Are you able to see it?

File: docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+![](<media/prereq-step4.png>)
+
+**Step 5.** In the PartsUnlimited solution, right-click on the PartsUnlimitedWebsite project and select "Add Application Insights telemetry."
+
+![](<media/prereq-step5.png>)
+
+**Step 6.** In the "Add Application Insights to Project" window, click on the "Use different account" hyperlink.
+
+![](<media/prereq-step6.png>)
```
Comment (EMaher) [inline docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md]: This image has your email displayed plainly.  Please blur it so you don't get a bunch of random emails.

File: docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+
+**Step 7.** Since we have the subscription and resource information for our PartsUnlimited Application Insights telemetry, click on the button, "Use advanced mode." 
+
+![](<media/prereq-step7.png>)
```
Comment (EMaher) [inline docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md]: This image has your email displayed plainly. Please blur it so you don't get a bunch of random emails.

File: docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+
+1. Set up Application Insights for PartsUnlimited
+
+2. Using Application Performance Monitoring to resolve performance issues
+
+###Task 1: Set up Application Insights for PartsUnlimited
+**Step 1.** In an Internet browser, navigate to <http://portal.azure.com> and
+sign in with your credentials.
+
+![](<media/prereq-step1.png>)
```
Comment (EMaher) [inline docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md]: A lot of these images display your full name.  I would suggest blurring that.  (Optional comment only.  Not necessary for approval.)

File: docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+![](<media/step1.png>) 
+
+**Step 12.** Return to the Application Insights performance monitoring view in the Azure Preview Portal and refresh the page. 
+
+![](<media/step12.png>) 
+
+In this lab, you learned how to set up Application Insights telemetry, and drill down into performance
+monitoring data through Application Insights in the new Azure Portal.
+
+Try these labs out for next steps:
```
Comment (EMaher) [inline docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md]: I assume these are upcoming HOLs?  I suggest adding a 'coming soon' comment to the ones not written yet.  I realize you will have to add the link to the TIP HOL after I approve the other pull request.

File: docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md (hunk 1 above)
Comment (EMaher) [inline docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md]: I see the change.  Thanks.

**Comment by EMaher** [comment]
Overall, looks good.  Just some minor comments to address.  Thanks.

File: docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md (hunk 3 above)
Comment (sawpresto) [inline docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md]: Done!

File: docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md (hunk 4 above)
Comment (sawpresto) [inline docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md]: Done!

File: docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md (hunk 5 above)
Comment (sawpresto) [inline docs/HOL_PartsUnlimited_Application_Performance_Monitoring/HOL_PartsUnlimited_Application_Performance_Monitoring.md]: Done!

**Comment by sawpresto** [comment]
Great, thanks! Pushed up another commit to blur out my name, good call. :)

**Comment by EMaher** [comment]
David asked for an explicit line in the intro to say that this HOL is created for the aspnet45 branch.  Once you add that I'll merge the pull request.

**Comment by sawpresto** [comment]
Sounds good, just added an explicit line in intro.

**Pull request merged**